### DISCUSSION TITLE
Tabi for project documentation?

### DISCUSSION THREAD

**Discussion by Almost-Senseless-Coder** [welpo/tabi#287] [Q&A]
Tabi is the best Zola theme I've encountered by far, but it aims to be a theme for a personal website/blog.

I need to provide documentation for an app I'm going to release soon and looked into themes designed for documentation:

- AdiDoks,
- Book,
- Docs as code,
- EasyDocs,
- Karzok

Many have accessibility issues which I'd need to fix first, as I know for a fact blind and vision impaired people will need to check the documentation out; with others, the multi language support is at least dubious and many don't seem to receive active development or even support anymore.

So I'm wondering: Has anyone tried using Tabi for more traditional (less blog-like) documentation? Having a blog is good for announcing new releases, events, etc. - but I feel a bit unsure whether my (not too tech-savvy) target audience would cope well with a blog-style documentation of the app itself.

**Comment by welpo** [comment]
Hey Tim!

I don't know if anyone's using tabi for documentation, but I doubt it. The layout isn't ideal

When choosing how to set up the documentation for [git-sumi](https://github.com/welpo/git-sumi) and [dōteki](https://github.com/welpo/doteki), I too looked at the available Zola themes for docs.

AdiDoks seemed promising, but its last commit was two years ago. I knew if I chose it, I would have to spend hours tinkering with it.

In the end, I went with [Docusaurus](https://github.com/facebook/docusaurus). It's not Zola, but it works without JS, offers multi-language support (I haven't tried it) and, as far as I can tell, it's accessible. The learning curve wasn't too steep.

Hope that helps!
